docs/concepts/k8s/function-ingress.md

```diff
@@ -17,11 +17,20 @@ In addition to configuring a service, Nuclio creates a [Kubernetes ingress](http
 
 ## Setting up an ingress controller
 
-In this guide, you'll set up a [Træfik](https://docs.traefik.io/) controller, but any type of Kubernetes ingress controller should work. You can read [Træfik's excellent documentation](https://docs.traefik.io/user-guide/kubernetes/), but for the purposes of this guide you can simply run the following commands to set up the controller:
+In this guide, you'll set up a [Træfik](https://docs.traefik.io/) controller, but any type of Kubernetes ingress controller should work. You can read [Træfik's excellent documentation](https://docs.traefik.io/user-guides/crd-acme/), but for the purposes of this guide you can simply run the following 
+commands to set up the controller in one of two ways:
```

Comment:
1. Join the two source lines (as was previously the case).
2. This isn't related to this PR, but "Træfik" (or "TrÃ¦fik" as it appears when I open the file in gVim) doesn't render currently with GitHub Markdown and it's also not how the product name is spelt [on their site](https://docs.traefik.io). It should be "Traefik".

=> I've attached an edited version with these changes in a newer PR comment.

Reply:
That line break has no effect on rendered document and it's there for maintainability